### PR TITLE
update go mod

### PR DESCRIPTION
**Pull request by yifeijin**
# Description
This PR updates go.mod file to latest dependencies and go mod tidy.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Review by bharathsreekanth — COMMENTED** [review]
What is the reason to do this update now?

**Review by nb950 — COMMENTED** [review]
please see blackduck scan issues with these latest versiosns
 for example viper 1.9.0 has high CVE flagged

**Review by bharathsreekanth — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -21,7 +21,7 @@ require (
 	github.com/cucumber/godog v0.12.1
 	github.com/dell/dell-csi-extensions/podmon v1.0.0
 	github.com/dell/dell-csi-extensions/volumeGroupSnapshot v0.0.0-20211104171507-7c7d0b6e3d35
-	github.com/dell/gocsi v1.3.1
+	github.com/dell/gocsi v1.4.0
```
Comment (JacobGros) [inline go.mod:24]: if latest gocsi is in main branch, but not released yet, can you uncomment the replace statement so we build and test with it in pipelines?

Comment (yifeijin) [reply go.mod:24]: 1.4 is already released. They are going to release 1.4.1 this quarter

**Review by nb950 — APPROVED** [review]

**Pull request merged**